sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -7,6 +7,10 @@ public partial interface ITableEntity
         string RowKey { get; set; }
         System.DateTimeOffset? Timestamp { get; set; }
     }
+    public static partial class QueryFilter
```

Comment:
Is this type in Search also called QueryFilter? Should we disambiguate with Tables prefix?

Reply:
It's called `SearchFilter`. Maybe `TablesFilter` is better?

Reply:
Yeah, stop taking all the good names :)